src/main/java/uk/gov/hmcts/ccd/domain/model/aggregated/CaseViewField.java

```diff
@@ -133,4 +135,10 @@ public Integer getShowSummaryContentOption() {
     public void setShowSummaryContentOption(Integer showSummaryContentOption) {
         this.showSummaryContentOption = showSummaryContentOption;
     }
+
+    @JsonIgnore
+    @Override
+    public String toString() {
+        return ReflectionToStringBuilder.toString(this);
```

Comment:
we had a discussion a while ago and decided not to use ReflectionToStringBuilder because it could hurt performance.
we decided to go for 
org.apache.commons.lang3.builder.ToStringBuilder

Reply:
ok will change to ToStringBuilder.reflectionToString(this, ToStringStyle.SHORT_PREFIX_STYLE)

Reply:
Actually I believe we went with Lombok.

See @ToString all over the code.

Reply:
Sure Andrew

Reply:
Done